### ISSUE TITLE
Clone note option

### ISSUE THREAD

**Issue by woonaval** [Dullage/flatnotes#47]
Hi,
I have a note which I use as a template to create other notes from it, and it would be great if there was a "Clone note" option, so I don't have to copypaste the template contents to a new note. I know it's not a very important thing 😅 but would be nice to have.
Thanks!!

**Comment by SmartPhoneLover** [comment]
Or maybe, instead of cloning include an option to use templates creating within Flatnotes. These templates may not be listed (indexed).

**Comment by krfdoc** [comment]
I agree.  We can copy on the backend .md file of course, but this would be very handy.  Love your project!
